spec/acts_as_revisionable_spec.rb

```diff
@@ -75,7 +75,9 @@ class RevisionableTestModel < ActiveRecord::Base
 
       attr_protected :secret
```

Comment:
It looks like `attr_protected` was removed from AR a couple of versions back. This caused a few spec failures for me.